record_web/lib/record_web.dart

```diff
@@ -145,15 +162,34 @@ class RecordPluginWeb extends RecordPlatform {
     return devices;
   }
 
+  double getMaxAmplitude() {
+    if(analyser == null) return minmumAmplitude;
+    analyser?.getFloatFrequencyData(amplitudeDataArray!);
+    var maxAmplitude = amplitudeDataArray!.reduce(max);
+    return maxAmplitude;
+  }
+
   void _onStart(html.MediaStream stream) {
     _mediaRecorder = html.MediaRecorder(stream);
     _mediaRecorder?.addEventListener('dataavailable', _onDataAvailable);
     _mediaRecorder?.addEventListener('stop', _onStop);
     _mediaRecorder?.start();
-
+    createAudioContext(stream);
     _updateState(RecordState.record);
   }
 
+  createAudioContext(html.MediaStream stream) {
+    audioCtx = AudioContext();
+    var source = audioCtx!.createMediaStreamSource(stream);
+    analyser = audioCtx!.createAnalyser();
+    analyser?.minDecibels = -90;
+    analyser?.maxDecibels = -10;
+    source.connect(analyser!);
+    analyser?.fftSize = 256;
```

Comment:
fftSize default is set 2048 according to https://developer.mozilla.org/en-US/docs/Web/API/AnalyserNode/fftSize
So why setting it to 256?

Reply:
to match amplitude values got from android and ios

Reply:
I experimented with it for some time and this was the best values to match with android and ios values
for consistency